### PR TITLE
Psql15

### PR DESCRIPTION
**Pull request by jburel**
This PR
* fixes warning and error
* adds support for psql 15
* removes psql 11 and 9.6

### CODE REVIEW

**Comment by jburel** [comment]
I will need to rebase this PR first

**Comment by pwalczysko** [comment]
Maybe problems with syntax during rebase ?
Screenshot 2024-02-21 at 14 26 12

**Review by pwalczysko — CHANGES_REQUESTED** [review]
There are some remnants of a rebase I believe 

``<<<<<<< HEAD`` in the redhat.yml file

The test is failing accordingly

**Comment by jburel** [comment]
I forgot to save the file!!

File: meta/main.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ galaxy_info:
   description: Upstream PostgreSQL client
   company: Open Microscopy Environment
   license: BSD
-  min_ansible_version: 2.6
+  min_ansible_version: '2.6'
```
Comment (pwalczysko) [inline meta/main.yml]: 2.10 as discussed ?
similarly to
https://github.com/ome/ansible-role-fluentd/blob/d1a5c0e19bdccdb0c8ae17f44c8f136bb449477e/meta/main.yml#L7 - according

File: vars/main.yml [hunk 2]
```diff
@@ -24,3 +16,7 @@ postgresql_distribution_redhat:
     repo: pgdg14
     basename: postgresql14
     setup: postgresql-14-setup
+  "15":
+    repo: pgdg15
+    basename: postgresql15
+    setup: postgresql-15-setup
```
Comment (pwalczysko) [inline vars/main.yml:22]: missing "16" ?

**Review by pwalczysko — CHANGES_REQUESTED** [review]
Two issues
- minimal ansible version should be 2.10
- a "16" block is missing in ``vars/main.yml``

**Review by pwalczysko — APPROVED** [review]

**Pull request merged**